clientbase/common.go

```diff
@@ -277,7 +277,7 @@ func NewAPIClient(opts *ClientOpts) (APIBaseClient, error) {
 	result.Ops = &APIOperations{
 		Opts:   opts,
 		Client: client,
-		Dialer: &websocket.Dialer{},
+		Dialer: &websocket.Dialer{HandshakeTimeout: 10 * time.Second},
```

Comment:
nit: 
Is 10 seconds enough? 
https://godoc.org/github.com/gorilla/websocket#pkg-variables
uses 45 seconds